src/field.js

```diff
@@ -1,14 +1,15 @@
 class Field {
   constructor(type, options = {}) {
-    this.name = ""
+    this.name = ''
     this.type = type
     this.options = options
+    this.isId = !!options.isId
```

Comment:
Should it be replicated?
if `options.isID` is `true` the object will have `meta.options.isID` and `meta.isID` fields. Is it necessary?

Reply:
Yes, should.
Because validation method inside `baseEntity.js` access the `meta.schema` not the options.

Reply:
is it possible to have:
```js
if (options && options.exceptIDs && definition.options.isId) continue
```
?